src/functions-and-arrays.js

```diff
@@ -1,73 +1,597 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else {
+    return num2;
+  }
+}
 
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot",
+];
+
+function findLongestWord(array) {
+  let longestWord = "";
+  if (!array.length) {
+    return null;
+  }
+
+  for (let i = 0; i < array.length; i++) {
+    if (longestWord.length < array[i].length) {
+      longestWord = array[i];
+    } else {
```

Comment:
Good job Nina! You don't need the `else{...}` here because there's no action defined in its scope and the function will keep working fine with just the "if" statement